src/helpers/user.js

```diff
@@ -1,15 +1,26 @@
 import ISO6391 from 'iso-639-1';
 import countries from 'svg-country-flags/countries.json';
 import { overwriteProfileDefaults } from '../utils/overwriteProfileDefaults';
+import { getUserFromLocalStorage } from '../api';
 
 export function isMentor(user) {
-  return user && user.roles.includes('Mentor');
+  return true;
+  // return user && user.roles.includes('Mentor');
 }
 
 export function isAdmin(user) {
   return user && user.roles.includes('Admin');
 }
 
+export function isMentorAvailable(){
```

Comment:
Are we need special function for `available` property? It's not different than the properties (such as name and description. The 2 function above are a bit different and basically, cover to a bit ugly check (imagine that in any place in the code we will write: `user && user.roles.includes('Admin')`. Also pay attention that they are stateless function so they get the user from outside, they don't know how to get it by themselves.

Reply:
I will remove this.